src/app.rs

```diff
@@ -86,7 +85,7 @@ pub fn start(mut conf: Config) {
                 output_folder.push(clone.key);
                 match OpenOptions::new().write(true).append(true).create(true).open(&output_folder)
                     .and_then(|mut file| {
-                        file.write(&format!("{} {}", cur_time, &result).as_bytes()[..])
+                        file.write(&format!("{:?} {}", cur_time, &result).as_bytes()[..])
```

Comment:
Oh.. Instant is defined purely as a precise monotonic clock. I'll update PR this evening to use [SystemTime](https://doc.rust-lang.org/std/time/struct.SystemTime.html) with [UNIX_EPOCH](https://doc.rust-lang.org/std/time/constant.UNIX_EPOCH.html)